tests/test_utils.py

```diff
@@ -32,6 +32,7 @@
     extract_model_from_parallel,
     find_device,
     listify,
+    operations,
```

Comment:
Why are we importing the entire operations here? Just import `send_to_device` and `convert_to_fp32` please